sdk/eventhubs/azure-messaging-eventhubs-stress/src/main/java/com/azure/messaging/eventhubs/stress/config/RateMeterConfig.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.stress.config;
+
+import com.azure.messaging.eventhubs.stress.util.Constants;
+import com.azure.messaging.eventhubs.stress.util.ScenarioOptions;
+import com.microsoft.applicationinsights.TelemetryClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.time.Duration;
+
+@Configuration
+public class RateMeterConfig {
+    @Autowired
+    private TelemetryClient telemetryClient;
+
+    @Autowired
+    private ScenarioOptions options;
+
+    @Bean
+    public RateMeter createRateMeter() {
```

Comment:
nit: the method name will be taken as the bean name, so it's better to use a noun here.

Reply:
Good suggestion. I will use rateMeter() instead.